.travis.yml

```diff
@@ -78,6 +78,6 @@ jobs:
 
     - name: "Release"
       stage: release
-      if: commit_message ~= /\[release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
+      if: commit_message ~= /\[release\]/ AND branch ~= /^(master|ATS-0000_refactor_packages|SP\/.+|HF\/.+)$/
```

Comment:
Needs updating before merge?

Reply:
Yes, but not the end of the world if missed.